doc: Add TTOG (#476)
TheThingsIndustries/lorawan-stack-docs: doc/content/gateways/thethingsoutdoor/_index.md

{{< figure src="GUI.png" alt="TTOG GUI">}}

Login with the username and password: `admin` / and the password from the label on the back of your unit (older firmware versions used the password `admin`). Make sure to update the password the first time you login.

Connect the gateway to {{% tts %}} via: **Packet Forward** > **Module 1 Settings** and enter the following settings:
 - **Server Address:** The address of your {{% tts %}} instance. See [Server Addresses]({{< ref "getting-started/server-addresses" >}}) 
 - **Server Uplink Port:** 1700
 - **Server Downlink Port:** 1700

{{< figure src="gateway-info.png" alt="Gateway Info">}}

Save the **Gateway ID**, you will need this later when adding the gateway in {{% tts %}}.

Set the network settings via **Network** > **WAN**. You can choose to connect via Ethernet or 3G/LTE. Reboot the gateway after configuring the network settings.

{{< note >}}If you connect via 3G/LTE, it will take a few minutes before the gateway will start routing messages.
{{</ note >}}

{{< note >}}

> To open the ODU GUI again, you can access the web interface via the fallback IP address: [192.168.11.10](http://192.168.11.10).

Reviewer: ```suggestion
To open the ODU GUI again, you can access the web interface via the fallback IP address: [`192.168.11.10`](http://192.168.11.10).
```

You will need to set your PC's IP address in the same range, e.g.:
- IP address: `192.168.11.100`
- Subnet Mask: `255.255.255.0`
- Router: `192.168.11.10`
{{</ note >}}
 
## Registration

Create a gateway in {{% tts %}} by following the instructions for [Adding Gateways]({{< ref "/gateways/adding-gateways" >}}).

Give your gateway a unique **Gateway ID** in {{% tts %}} - this is just for you to identify the gateway and it does not need to be the same as the **Gateway ID** displayed in the gateway.

> The **Gateway ID** in the gateway is used to derive the EUI, which can be found in the ODE GUI, under **Packet Forward** > **Module 1 Settings**. Add `0000` at the beginning of the Gateway ID to get the Gateway EUI. For example `80029c10xxxx` becomes `000080029c10xxxx`.

Reviewer: ```suggestion
The **Gateway ID** in the gateway is used to derive the EUI, which can be found in the ODE GUI, under **Packet Forward** > **Module 1 Settings**. Add `0000` at the beginning of the Gateway ID to get the Gateway EUI. For example, the Gateway ID `80029c10xxxx` will yield the Gateway EUI `000080029c10xxxx`.
```

> Add this EUI in {{% tts %}} as **Gateway EUI**.

Reviewer: ```suggestion
Add this EUI in {{% tts %}} as a **Gateway EUI**.
```

> Once you have registered your gateway, if all the steps have been followed correctly, your gateway will now connect to {{% tts %}}.

Reviewer: ```suggestion
If all steps have been followed correctly, once you register your gateway it will connect to {{% tts %}}.
```

## Troubleshooting

> * When you cannot find a way to change the LoRaWAN settings, you might need to update the System Firmware. You can do so via:
>  * **System** > **System Firmware** 
>  * Download [this file](https://connectedthings.store/files/WAPS-232N_LW_OPDK_GUI_1.01.09_3416004962.tar.gz), making sure to keep the file name exactly the same (don't try to unzip this file)
>  * Upload the file to and click **Upgrade**
> 
>  {{< figure src="upgrade_1.png" alt="Upgrade Page">}}
> 
>  * The gateway will automatically reboot with the upgraded firmware

Reviewer: ```suggestion
When you cannot find a way to change the LoRaWAN settings, you might need to update the System Firmware. You can do so by following steps below.
* Navigate to **System** > **System Firmware** on the left hand menu
* Download [this file](https://connectedthings.store/files/WAPS-232N_LW_OPDK_GUI_1.01.09_3416004962.tar.gz), making sure to keep the file name exactly the same (do not unzip this file)
* Upload the file and click **Upgrade**

{{< figure src="upgrade_1.png" alt="Upgrade Page">}}

* The gateway will automatically reboot with the upgraded firmware
```
